### PR TITLE
[Kotlin Migration] Lookup

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
CI failure

File: AnkiDroid/src/main/java/com/ichi2/anki/Lookup.kt [hunk 1]
```diff
@@ -1,192 +1,202 @@
...
-            case DICTIONARY_NONE:
-            default:
-                mIsDictionaryAvailable = false;
-                break;
+    private const val DICTIONARY_NONE = 0 // use no dictionary
+    private const val DICTIONARY_AEDICT = 1 // Japanese dictionary
+    private const val DICTIONARY_EIJIRO_WEB = 2 // japanese web dictionary
+    private const val DICTIONARY_LEO_WEB = 3 // German web dictionary for English, French, Spanish, Italian,
+
+    // Chinese, Russian
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/Lookup.kt:23]: nit/Cleanup: comments were changed, ideally turn into doccomment

File: AnkiDroid/src/main/java/com/ichi2/anki/Lookup.kt [hunk 2]
```diff
@@ -1,192 +1,202 @@
...
+    private const val DICTIONARY_LEO_APP = 4 // German web dictionary for English, French, Spanish, Italian,
+
+    // Chinese, Russian
+    private const val DICTIONARY_COLORDICT = 5
+    private const val DICTIONARY_FORA = 6
+    private const val DICTIONARY_NCIKU_WEB = 7 // chinese web dictionary
+    private var mContext: Context? = null
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/Lookup.kt:30]: Cleanup: lateinit

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
few nits, should be good to go after

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]

**Pull request merged**